lib/auth/api.go

```diff
@@ -721,6 +721,76 @@ type DiscoveryAccessPoint interface {
 	DeleteDatabase(ctx context.Context, name string) error
 }
 
+// ReadOktaAccessPoint is a read only API interface to be
+// used by an Okta component.
+//
+// NOTE: This interface must match the resources replicated in cache.ForOkta except for
```

Comment:
I'm confused -- looking at the other PR, `ForOkta` appears to return a `Config` object. The relationship between this interface and that function is unclear.

Reply:
I updated the comment -- let me know if the current version is clearer.